modules/iform_licences/iform_licences.module

```diff
@@ -163,7 +163,7 @@ function iform_licences_user_login (&$edit, $account) {
     ));
     $currentMediaLicenceId = $current[0]['media_licence_id'];
     if (!isset($currentMediaLicenceId)) {
-      drupal_set_message(t('You have not yet specified a media licence. This limits the usefulness of your images. <a href="user/@userid/edit#licenceopts">To set a licence, edit your profile page, and select an option.</a>', ['@userid' => $user->uid]));
+      drupal_set_message(t('You have not yet specified a media licence. This limits the usefulness of your images. <a href="/user/@userid/edit#licenceopts">To set a licence, edit your profile page, and select an option.</a>', ['@userid' => $user->uid]));
```

Comment:
Since this is Drupal 7, it's possible that dirty URLs will be in use (http://www.example.com?q=user for example). I think this change will break in that scenario? This might be helpful: https://api.drupal.org/api/drupal/includes!common.inc/function/url/7.x